src/components/instructor/AssignmentsView.js

```diff
@@ -9,16 +11,83 @@ import {useLocation} from 'react-router-dom'
 // display a table with columns 
 // assignment id, title, dueDate and buttons to grade, edit, delete each assignment
 
-const AssignmentsView = (props) => {
```

Comment:
This shouldn't be deleted

Reply:
This is required for the functionality of the file